pkgs/racket-doc/scribblings/reference/stx-ops.scrbl

```diff
@@ -370,14 +371,14 @@ at @tech{phase level} 0 are shifted to the @tech{label phase level}.
 If @racket[shift] is @racket[0], then the result is @racket[stx].}
 
 
-@defproc[(generate-temporaries [stx-pair (or syntax? list?)]) 
+@defproc[(generate-temporaries [v stx-list?])
          (listof identifier?)]{
 
 Returns a list of identifiers that are distinct from all other
 identifiers. The list contains as many identifiers as
-@racket[stx-pair] contains elements. The @racket[stx-pair] argument
-must be a syntax pair that can be flattened into a list. The elements
-of @racket[stx-pair] can be anything, but string, symbol, keyword
+@racket[v] contains elements. The @racket[v] argument
+must be a value that can be flattened into a list (satisfying @racket[stx-list?]).
```

Comment:
Since the corrected contract already says that `v` must be `stx-list?`, should this sentence be dropped?

Reply:
That makes sense. I dropped the sentence.